code_samples/simple_language/src/main/java/com/intellij/sdk/language/SimpleReferenceContributor.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package com.intellij.sdk.language;
+
+import com.intellij.openapi.util.TextRange;
+import com.intellij.patterns.PlatformPatterns;
+import com.intellij.psi.*;
+import com.intellij.util.ProcessingContext;
+import org.jetbrains.annotations.NotNull;
+
+import static com.intellij.sdk.language.SimpleAnnotator.*;
+
+public class SimpleReferenceContributor extends PsiReferenceContributor {
+  @Override
+  public void registerReferenceProviders(@NotNull PsiReferenceRegistrar registrar) {
+    registrar.registerReferenceProvider( PlatformPatterns.psiElement( PsiLiteralExpression.class ),
```

Comment:
please reformat to have less indentation in `PsiReferenceProvider` implementation